docs/settings.rst

```diff
@@ -269,6 +269,21 @@ Either don't ``chroot`` on these systems or set the 'Type' of the
 service to 'simple' instead of 'notify' (refer to the systemd
 documentation on how to modify unit-files).
 
+.. _setting-compare-signatures-on-zone-freshness-check:
+
+``compare-signatures-on-zone-freshness-check``
+----------------------------------------------
+
+.. versionadded:: 4.7.0
+
+-  Boolean
+-  Default: yes
+
+Turning this off will disable the DO flag for SOA queries during zone freshness checks of secondary zones
+to work around truncated SOA responses. It will also disable signature comparison which is used to detect
+signature changes even when the serial was not increased. Hence, disable this setting only if the Primary
```

Comment:
```suggestion
to work around truncated SOA responses. It will also disable signature comparison which is used to detect
signature changes without changes to the serial. Hence, disable this setting only if the Primary
```